docs/personalization/integrate_recommendation_service.md

```diff
@@ -151,6 +151,62 @@ if ($recommendations && isset($recommendations->recommendationResponseList)) {
 curl_close($curl);
 ```
 
+## Send emails with recommendations
+
+By using [email triggers]([[= user_doc =]]/personalization/triggers.md), your organization can send emails to individual visitors, for example, to invite them to return to the website or remind them of items abandoned in a cart.
```

Comment:
The link is broken. Only visitors? Clients, customers too. I would change the order in the sentence to more active and calling users to action.

Reply:
Agreed re. order, while "client" and "customers" are used in a different meaning throughout perso:
"Set up customer credentials", "PERSONALIZATION_CUSTOMER_ID" refer to Ibexa's customers
"client side", "client-server architecture" refer to Personalization client
"visitor" is used consistently to refer to people who browse and buy